frontend/packages/ux-editor/src/components/leftMenu/LeftMenu.tsx

```diff
@@ -68,26 +69,30 @@ export const LeftMenu = ({ className }: LeftMenuProps) => {
 
   return (
     <div className={cn(className, classes.rightMenu)}>
-      <Accordion color='subtle'>
-        {shouldDisplayFeature('configureLayoutSet') && (
-          <Accordion.Item defaultOpen={layoutSetNames?.length > 0}>
-            <Accordion.Header>{t('left_menu.layout_sets')}</Accordion.Header>
-            <Accordion.Content>
-              {layoutSetNames ? (
-                <>
+      {shouldDisplayFeature('configureLayoutSet') &&
+        (layoutSetNames ? (
+          <>
+            <div className={classes.dropDownContainer}>
+              <div className={classes.selectLabelAndIcon}>
+                <span>{t('left_menu.layout_dropdown_menu_label')}</span>
```

Comment:
Det er en egen [label](https://www.w3schools.com/tags/tag_label.asp)-tagg som skal brukes til dette. Den gjør at man får fokus på `select`-elementet når man klikker på den. Bare vær oppmerksom på at man i React bruker `htmlFor` til å referere til komponenten, i stedet for `for`, som man bruker i ren HTML.